docs/getting_started.rst

```diff
@@ -14,13 +14,23 @@ Install with Conda
 ------------------
 .. _install-with-conda:
 
-`conda <http://www.continuum.io/blog/conda>`_ is a python package manager built for scientific python. Unlike ``easy_install`` or ``pip``, it handles binaries and binary dependencies, which are critical for most scientific workflows. If you're a ``conda`` user, you can install OpenPathSampling by adding the omnia channel. If you're not a conda user, you should look into it. ::
+We recommend using `conda` to install OpenPathSampling.  `conda
+<http://www.continuum.io/blog/conda>`_ is a python package manager built for
+scientific python. Unlike ``easy_install`` or ``pip``, it handles binaries
+and binary dependencies, which are critical for most scientific workflows.
+If you're a ``conda`` user, you can install OpenPathSampling by adding the
+omnia channel. ::
 
-To install OpenPathSampling with conda, use the following commands ::
+To install the most recent release of OpenPathSampling with conda, use the
+following commands ::
 
   $ conda config --add channels http://conda.binstar.org/omnia
```

Comment:
Also, you're missing some dependencies in your conda `meta.yaml`.  After installation on `osx`:
```
>>> import openpathsampling
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/choderaj/miniconda/lib/python2.7/site-packages/openpathsampling-0.0.1a0-py2.7.egg/openpathsampling/__init__.py", line 69, in <module>
  File "/Users/choderaj/miniconda/lib/python2.7/site-packages/openpathsampling-0.0.1a0-py2.7.egg/openpathsampling/analysis/__init__.py", line 1, in <module>
  File "/Users/choderaj/miniconda/lib/python2.7/site-packages/openpathsampling-0.0.1a0-py2.7.egg/openpathsampling/analysis/histogram.py", line 3, in <module>
ImportError: No module named matplotlib.pyplot
```